skaffold/thunder-setup.sh

```
@@ -0,0 +1,304 @@
+        case "$http_code" in 2*) ;; *) echo "ERROR: Update (after APP-1020) failed for ${client_id} (HTTP ${http_code}): ${body}"; exit 1 ;; esac
+      fi
+    else
+      case "$http_code" in 2*) ;; *) echo "ERROR: Create failed for ${client_id} (HTTP ${http_code}): ${body}"; exit 1 ;; esac
+    fi
+    LAST_APP_ID=$(echo "$body" | python3 -c "import sys,json; d=json.load(sys.stdin); print(d.get('id',''))" 2>/dev/null || echo "")
+  fi
+  echo "  ✓ ${client_id}"
+}
+
+thunder_confidential() {
+  local name="$1" desc="$2" cid="$3" secret="$4"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)